packages/config/scripts/tokens/add/index.ts

```diff
@@ -0,0 +1,78 @@
+import { EthereumAddress } from '@l2beat/shared'
+import chalk from 'chalk'
+import { config as dotenv } from 'dotenv'
+import { providers } from 'ethers'
+import { writeFileSync } from 'fs'
+import { z } from 'zod'
+
+import { tokenList } from '../../../src'
+import { TokenInfo } from '../../../src/tokens/types'
+import { getEnv } from '../../checkVerifiedContracts/utils'
+import { getTokenInfo } from './getTokenInfo'
+
+async function main() {
+  const [address, category] = handleCLIParameters()
+  if (!address || !category) {
```

Comment:
You can probably move this check to handleCLIParameters and return non undefined variables from it

Reply:
doing it this way allows me to handle this kind of error with human readable console.log, if I throw then the error will be less readable